src/test/java/ScrabbleGameTest.java

```diff
@@ -1,13 +1,15 @@
 import org.junit.jupiter.api.Test;
-import org.mockito.Mock;
 import org.mockito.Mockito;
-
 import static org.junit.jupiter.api.Assertions.*;
 
 class ScrabbleGameTest {
     ScrabbleDictionary dictionary = Mockito.mock(ScrabbleDictionary.class);
     LetterPool letterPool = Mockito.mock(LetterPool.class);
 
+    private final String SUCCESS = "Success!";
+    private final String NOT_A_WORD = "Not a word";
+    private final String NO_TILES = "Tiles not available";
```

Comment:
Move these Strings into the class instead of the tests, make them static and use them in both the class and the tests